treccovid_semantic_search/workload.json

```diff
@@ -0,0 +1,32 @@
+{% import "benchmark.helpers" as benchmark with context %}
+
+{
+  "version": 2,
+    "description": "Trec-Covid is a dataset collection of documents about COVID-19 information.",
+  "indices": [
+    {
+      "name": "treccovid",
+      "body": "index.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "treccovid",
```

Comment:
Is the default trec-covid dataset big enough to give reliable information about latency? It has about 170K documents. We are going to host the corpora anyways, can we pre-process it and increase the number of documents by creating multiple copies of the same doc? Like 6 copies, this should give us about 1M.

Reply:
1M documents works for me and it would also solve the dataset hosting issue, but the queries should be hosted in the same place.

Reply:
I think more on this option, probably it's better to make it an enhancement to this workload or even a separate workload. I think we are good for this PR with what we have